File: API/API/Entities/Dish.cs
namespace API.Entities;

public class Dish
{
    public int Id { get; set; }
    public string Name { get; set; }
    public TimeOnly CookingTime { get; set; }
    public string YouWillNeed { get; set; }
    public decimal Price { get; set; }
    public bool IsAvailableForSingleOrder { get; set; }

    public ICollection<Image> Images { get; set; }
    public ICollection<DishIngredient> Ingredients { get; set; }
}

File: client/src/app/core/models/dish.ts
import { Image } from "./image";
import { Ingredient } from "./ingredient";

export interface Dish {
    id: number;
    name: string;
    cookingTimeHours: number;
    cookingTimeMinutes: number;
    youWillNeed: string;
    price: number;
    isAvailableForSingleOrder: boolean;
    images: Image[];
    ingredients: Ingredient[];
}